erizo/src/erizo/MediaStream.cpp

```diff
@@ -732,6 +745,11 @@ uint32_t MediaStream::getTargetVideoBitrate() {
   uint32_t bitrate_from_max_quality_layer = getBitrateFromMaxQualityLayer();
 
   uint32_t target_bitrate = max_bitrate;
+
+  if (target_is_max_video_bw_) {
```

Comment:
could max_bitrate be 0 because it has not been initialized? Maybe we should return kInitialBitrate in that case, as we do below.